test/dlc_tests/benchmark/sagemaker/tensorflow/training/test_trcomp_performance.py

```diff
@@ -175,3 +175,75 @@ def test_resnet101_at_fp16(self, instance_type, num_gpus, total_n_gpus, instance
+            f"runtime.num_gpus={num_gpus},"\
+            f"runtime.distribution_strategy={distribution_strategy},"\
+            f"runtime.mixed_precision_dtype=float16,"\
+            f"task.train_data.global_batch_size={batch},"\
+            f"task.train_data.cache={caching},"\
+            f"trainer.train_steps={train_steps},"\
+            f"trainer.steps_per_loop={steps_per_loop},"\
+            f"trainer.summary_interval={steps_per_loop},"\
+            f"trainer.checkpoint_interval={train_steps},"
+            estimator = TensorFlow(
+                                sagemaker_session=sagemaker_session,
+                                git_config={
+                                    'repo': 'https://github.com/huggingface/transformers.git',
+                                    'branch': 'v4.21-release',
+                                },
+                                source_dir='.',
+                                entry_point='examples/tensorflow/summarization/run_summarization.py',
+                                model_dir=False,
+                                instance_type=instance_type,
+                                instance_count=instance_count,
+                                image_uri=tensorflow_training,
+                                hyperparameters={
+                                    TrainingCompilerConfig.HP_ENABLE_COMPILER : True,
+                                    "model_name_or_path": "facebook/bart-base",
+                                    "dataset_name": "xsum",
+                                    "fp16": 1,
+                                    "num_train_epochs": 1,
+                                    "pad_to_max_length": True,
+                                    "do_train": True,
+                                    "do_eval": False,
+                                    "overwrite_output_dir": True,
+                                    "save_strategy": "no",
+                                    "logging_strategy": "no",
+                                    "evaluation_strategy": "no",
+                                },
+                                debugger_hook_config=None,
+                                disable_profiler=True,
+                                max_run=60*60*1, # Timeout in 1 hours
+                                base_job_name=f"tf{framework_version.replace('.','')}-trcomp-bench-facebook/bert-base",
+                                role="SageMakerRole",
+                            )
+            estimator.fit(logs=True, wait=True)
+            
+            captured = capsys.readouterr()
+            logs = captured.out + captured.err
+            match = re.search('Billable seconds: ([0-9]*)', logs)
+            billable = int(match.group(1))
+
+            short_version = '.'.join(framework_version.split('.')[:2])
+            threshold = TRCOMP_THRESHOLD['tensorflow'][short_version]['facebook/bert-base'][instance_type][instance_count][batch]
```

Comment:
please configure this variable. Currently, no entry exists for 2.10. configure these entries from local runs.